docs/specification/well-known-schema-formats.md

```diff
@@ -0,0 +1,33 @@
+## Well-known schema formats
+
+### proto
+
+- schema: .proto
+- schema_name: unspecified
+
+### proto-set
```

Comment:
rename to `proto` or something like `proto-binary-descriptor-set` or `proto.FileDescriptorSet.binary`?